app/src/components/governance/ParticipantList.vue

```diff
@@ -9,7 +9,13 @@
       >
         <div class="first-column">
           <span class="icon">
-            <img v-if="participant.picture" :src="participant.picture" />
+            <img
```

Comment:
wrong fix. the picture is not set in the API

Reply:
OK, let me fix this in API